RetailSuiteAccounting/Operation/15_ProvideAccountingData.md

```diff
@@ -0,0 +1,161 @@
+[!!User Interface export DATEV CSV](../UserInterface/03b_Export.md#datev-csv)  
+[!!User Interface GDPdU export](../UserInterface/03b_Export.md#gdpdu-export)  
+[!!User Interface CVS format definition](../UserInterface/03c_CSVFormatDef.md)  
+[!!User Interface BWA](../UserInterface/01_Book.md#bwa)  
+[!!User Interface BWA assignment](../UserInterface/02i_BWAAssignment.md)  
+[!!Manage the BWA reports](../Integration/07_ManageBWAReports.md)  
+
+
+
+# Provide the accounting data
+
+The *Accounting* module offers an export function to export data in different formats. The most widely used financial formats for accounting software are supported.
+
+Exporting data may be necessary to perform an external audit, create a balance sheet or prepare the annual accounts. These tasks are usually performed at the end of a fiscal year. The system also includes a reporting functionality to create different report types and export them in PDF format.
+
+
+## Export the data for DATEV
+
+Accounting data can be exported into a DATEV CSV compatible file to be later imported in DATEV software for further processing.  
+
+#### Prerequisites
+
+ - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+ - A fiscal year is selected, see [Select a fiscal year](./01_SelectFiscalYear.md).
+
+#### Procedure
+
+*Accounting > Select fiscal year > Extras > Tab EXPORT*
+
+![Export](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/Export.png "[Export]")
+
+1. Click the [DATEV CSV] button.    
+The *Export: DATEV CSV: Step 1* window is displayed.
+
+  ![Export DATEV CVS: Step 1](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportDatevCVS01.png "[Export DATEV CVS: Step 1]")
+
+2. Select the fiscal year to be exported in the *Fiscal year* drop-down list.
+
+3. Enter the months and journals to be exported in the *Period* field.
+
+4. Select the appropriate radio button (**Export all/All from last export on (if last export exists, otherwise all)**) in the *Selection* field. Mark the *Set marker to export time* checkbox, if necessary.
```

Comment:
```suggestion
4. Select the appropriate radio button in the *Selection* field. The following option are available:
  - **Export all**
  - **All from last export on (if last export exists, otherwise all)**    
 If desired, select the *Set marker to export time* checkbox.
```